demoapp/src/main/java/com/cube/fusion/android/demoapp/activity/ContentActivityImpl.kt

```diff
@@ -36,7 +37,10 @@ class ContentActivityImpl : FusionContentActivity() {
 	}
 
 	lateinit var binding: ActivityFusionImplBinding
-	override val contentBinding: ContentFragmentViewBinding get() = binding.pageContent
+	@Suppress("TYPE_MISMATCH") // Note: Android Studio incorrectly types this due to being part of the repo with the view in. This annotation would not be needed on an actual project.
+	override val contentBinding: ContentFragmentViewBinding by lazy { ContentFragmentViewBinding.bind(binding.pageContent) }
```

Comment:
Very tiny and doesn't need to change now but `lazy` uses double checked locking by default which can cause a bit of a performance hit, probably preferable to use `lazy(LazyThreadSafetyMode.NONE) {}`

Reply:
Good shout, I'll fix that

Reply:
Actually come to think about it, it probably makes more sense for these properties to be `lateinit` rather than `by lazy`

Reply:
Actually yeah, that's a better shout given the context!